classes/01class/exercises/c01-aws01/mendoncaj/README.md

```diff
@@ -0,0 +1,91 @@
+## EC2 (c01-aws01)
+
+- Use the CLI to create a new EC2 Instance inside the default VPC with a public IP address and connect to it though SSH.
+
+> **Public instance**
+```
+aws ec2 create-key-pair \
+--dry-run \
+--key-name academy \
+--tag-specifications 'ResourceType=key-pair,Tags=[{Key=ref,Value=devops-academy}]'
+--output text > academy.pem
+```
+
+```
+aws ec2 create-security-group \
+--group-name ssh-restrict \
+--description "access only through ssh" \
+--vpc-id vpc-076e78c1aecad75e2
+```
+
+```
+aws ec2 authorize-security-group-ingress \
+--group-name ssh-restrict \
+--protocol tcp \
+--port 22 \
+--cidr 0.0.0.0/0
```

Comment:
You are allowing all sources instead of only your IP, which is fine if that's what you intended.